support-config/src/main/scala/com/gu/monitoring/SafeLogger.scala

```diff
@@ -1,21 +1,33 @@
 package com.gu.monitoring
```

Comment:
this file just has a bit of a shuffle to make everything available from the trait rather than by acessing the static object.